Index: README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>Bluetooth Scanner Utility\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/README.md b/README.md
--- a/README.md	(revision 19ce5cd42eaba46443748f0fa08b34382cb01432)
+++ b/README.md	(date 1745668662968)
@@ -1,1 +1,1 @@
-Bluetooth Scanner Utility
+
